BossMod/Modules/RealmReborn/Trial/T09WhorleaterH/T09WhorleaterHBodyslam.cs

```diff
@@ -20,16 +20,14 @@ public override IEnumerable<Source> Sources(BossModule module, int slot, Actor a
         public override void Update(BossModule module)
         {
             base.Update(module);
-            var boss = module.Enemies(OID.Boss).FirstOrDefault();
-            if (boss != null)
             {
                 if (LeviathanZ == default)
-                    LeviathanZ = module.Enemies(OID.Boss).First().Position.Z;
-                if (boss.Position.Z != LeviathanZ && boss.Position.Z != 0)
+                    LeviathanZ = module.PrimaryActor.Position.Z;
+                if (module.PrimaryActor.Position.Z != LeviathanZ && module.PrimaryActor.Position.Z != 0)
```

Comment:
Does this belong here? Seems duplicate of leviathan PR?

Reply:
you are right, that was a merge error again like the 6.55 patch